### PR TITLE
[DEV-6662] Dark Released Global Nav for About the Data using Query Pararms

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**
Dark Release new url in DEV & QAT w/ query params for mobile and desktop.

**Technical details:**
FYSA: Having trouble mocking out query params, one test is failing

c285606 removing glossary from header …

da20724 refactoring dropdown and dropdownitem …

218c0dd New Badge Style clean up

0ebb0ae Dark Release Logic for Agency Submission Stats …

65446b6 Renaming url & using query params …

374b1e7 Mobile Nav Update …

**JIRA Ticket:**
[DEV-6662](https://federal-spending-transparency.atlassian.net/browse/DEV-6662)

**Mockup:**
Desktop: https://bahdigital.invisionapp.com/share/UHIAFZOV624
Mobile: https://bahdigital.invisionapp.com/share/5BIAFZQZSJM

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
`N/A` Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
`N/A` [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
`N/A` [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
`N/A` [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [x] Code review complete

### CODE REVIEW

File: src/js/components/sharedComponents/header/Dropdown.jsx [hunk 1]
```diff
@@ -16,88 +15,74 @@ const propTypes = {
...
+    label,
+    title,
+    items
+}) => {
+    const [expanded, setExpanded] = useState(false);
 
-        this.state = {
-            expanded: false
-        };
+    const clickedButton = () => {
```
Comment (jonhill13) [inline src/js/components/sharedComponents/header/Dropdown.jsx:25]: This can be `toggleMenu` and we can delete `expandMenu ` and `collapseMenu`

Comment (MaxwellKendall) [reply src/js/components/sharedComponents/header/Dropdown.jsx:25]: Not sure if we can actually. my changes aren't related to this but if we can improve w/o effecting UX I can change. it appears we need this to allow for expand/collapse on mouse enter/leave:

image

File: src/js/helpers/glossaryHelper.js [hunk 2]
```diff
@@ -15,3 +15,17 @@ export const fetchSearchResults = (params) => apiRequest({
     method: 'post',
     data: params
 });
+
+export const getNewUrlForGlossary = (pathname, url, queryParams = '') => {
```
Comment (jonhill13) [inline src/js/helpers/glossaryHelper.js:19]: Do we have comments explaining this function?

Do we have tests for this?

**Review by jonhill13 — CHANGES_REQUESTED** [review]

File: src/js/helpers/glossaryHelper.js (hunk 2 above)
Comment (MaxwellKendall) [reply src/js/helpers/glossaryHelper.js:19]: @jonhill13 writing them right now!

Comment (MaxwellKendall) [reply src/js/helpers/glossaryHelper.js:19]: @jonhill13 pushed tests.

**Review by jonhill13 — APPROVED** [review]

**Pull request merged**